run_sets/testsuite.yml

```diff
@@ -16,6 +16,7 @@
 - features/core_srv_users.feature
 - features/core_srv_credentials.feature
 - features/core_srv_create_activationkey.feature
+- features/core_proxy.feature
```

Comment:
maybe we could name this
```  features/core_proxy_if_enabled.feature```
 or add a comment above to say that this doesnt run always

Reply:
this comment is inside of the file

but the name could indeed be better

Speaking of which,

     core_setup_user_and_proxy.feature

has now become misleading (it's about HTTP proxy, not suma proxy)

Reply:
Remove the name, for that feature. The stuff inside the file, we need something outside, for the global picture when reading all feature tog

Reply:
Remove the name, for that feature. The stuff inside the file, we need something outside, for the global picture when reading all feature tog

Reply:
"Remove the name, for that feature." Which feature, the HTTPS proxy or the SUMA proxy one?

And remove how?

Reply:
```core_setup_user``` is fine

Reply:
Right, deserves a separate PR. I have no idea of correct name but i'll find one.

Reply:
done, with `core_first_settings` (it's not only about users, but also organization, http proxy, etc)